Drakhtar/GameObjects/Box.h

```diff
@@ -15,28 +18,86 @@ enum class TextureInd {
 
 class Box final : public GameObject {
  protected:
+  /**
+   * \brief The row and column this box occupies in the board matrix.
+   */
   Vector2D<int> boardIndex_;
+
+  /**
+   * \brief The unit currently placed in the box.
+   */
   Unit *content_;
+
+  /**
+   * \brief The index of current texture in the box texture array.
+   */
   TextureInd cellTexture_;
+
+  /**
+   * \brief Whether or not the mouse is over the box.
+   */
   bool hovered_ = false;
+
+  /**
+   * \brief Array of all textures de box may render.
+   */
   Texture *cellTextures_[5]{};
+
+  /**
+   * \brief Vector2D of the width and height of the box.
+   */
   const Vector2D<int> size_;
 
  public:
   Box(Scene *scene, const Vector2D<int> &pos, const Vector2D<int> &size,
       const Vector2D<int> &boardIndex, Unit *unit);
 
+  /**
+   * \return The SDL_Rect of the box, being its position, width and height.
```

Comment:
nit: 
```suggestion
   * \return The SDL_Rect of the box, being its position, width, and height.
```

Reply:
The absence of the comma is correct and intended.